packages/blaze/attrs.js

```diff
@@ -171,8 +171,14 @@ var BooleanHandler = AttributeHandler.extend({
   }
 });
 
+var origUpdate = AttributeHandler.prototype.update;
 var ValueHandler = AttributeHandler.extend({
   update: function (element, oldValue, value) {
+    var self = this;
+    var args = arguments;
+
+    origUpdate.apply(self, args);
```

Comment:
I'm wary of saving the `origUpdate` permanently like this. Can you just say `AttributeHandler.prototype.update.apply(this, arguments)` here?